addons/portal/static/src/scss/portal.scss

```diff
@@ -561,3 +574,108 @@ form label {
     }
 }
 
+// Pretty slider (checkbox)
+
+div.o_boolean_toggle.custom-control.custom-checkbox {
+    $line-height-computed: $line-height-base * $font-size-base;
+    $slider-width: $line-height-computed * 1.5;
+    $font-size: $font-size-base * 1.2;
+
+    display: inline-block;
+    padding-left: $slider-width + 0.25rem;
+
+    > label.custom-control-label {
+        > i {
+            color: white;
+            cursor: pointer;
+            font-size: $font-size;
+            position: absolute;
+            top: 0.175rem;
+            &.fa-times-circle {
+                right: 1.25rem;
+            }
+            &.fa-check-circle {
+                right: 0.5rem;
+            }
+        }
+        &::before,
+        &::after {
+            content: "";
+            top: 0;
+            left: -($slider-width + 0.25rem);
+        }
+        &::before {
+            width: $slider-width;
+            height: 100%;
+            background-color: $o-portal-checkbox-unchecked-bg-color !important;
+            border-radius: 100px;
+            outline: none !important;
+        }
+    }
+    > input.custom-control-input:checked + label.custom-control-label {
+        &::after {
+            background-image: none;
+        }
+        > i.fa-times-circle {
+            display: none !important;
+        }
+    }
+    > input.custom-control-input:checked + label.custom-control-label::before {
+        background-color: $o-brand-primary !important;
+    }
+    > input.custom-control-input:not(:checked) + label.custom-control-label {
+        > i.fa-check-circle {
+            display: none !important;
+        }
+    }
+}
+
+.custom-control.custom-checkbox .custom-control-input {
+    // Force default bts default border color
+    &:not(:checked):not(:indeterminate):not(:disabled) ~ .custom-control-label:before {
+        border-color: $o-portal-checkbox-checked-border;
+    }
+
+    &:disabled {
+        ~ .custom-control-label > i {
+                cursor: not-allowed;
+        }
+    }
+
+    @media print {
+        -webkit-print-color-adjust: exact !important;
+        color-adjust: exact !important;
+    }
+}
+
+.oe_grey {
+    opacity: 0.5;
+}
+
+.oe_bold {
+    font-weight: $font-weight-bold;
+}
+
+// Kanban-like cards view
+
+.oe_cart {
+    table td:first-child {
+        min-width: 76px;
+    }
+
+    > .oe_structure {
+        clear: both;
+    }
+}
+
+.js_change_shipping {
+    cursor: pointer;
+}
+
+.o-card-default-bg {
+    background-color: $gray-300 !important;
+}
+
+.o-card-default-bg-action:hover {
+    background-color: $gray-200 !important;
+}
```

Comment:
awa-odoo comment (todo)

You have to be extra careful when defining new css rules.

This means that anyone adding the "oe_gray" class to its own page will get your rule, same thing for oe_cart, etc.

Usually, when customizing page style, we try to prefix and namespace everything to avoid collision.

So you would have a "o_portal_sale_kanban_card" for example.
Usually it starts with "o", then the module/namespace, then the name of the actual thing it's referring to.